runtime/moonbase/src/lib.rs

```diff
@@ -665,26 +665,21 @@ impl InstanceFilter<Call> for ProxyType {
 	fn filter(&self, c: &Call) -> bool {
 		match self {
 			ProxyType::Any => true,
-			ProxyType::NonTransfer => matches!(
-				c,
-				Call::System(..) |
-				Call::Timestamp(..) |
-				Call::ParachainStaking(..) |
-				// Call::Session(..) |
-				Call::Democracy(..) |
-				Call::CouncilCollective(..) |
-				Call::TechComitteeCollective(..) |
-				// Call::Treasury(..) |
-				Call::Utility(..) |
-				Call::Scheduler(..) |
-				Call::Proxy(..)
-			),
+			ProxyType::NonTransfer => {
+				matches!(
+					c,
+					Call::System(..)
+						| Call::Timestamp(..) | Call::ParachainStaking(..)
```

Comment:
> The session and scheduler changes are good for me. The treasury one seems riskier. This could allow users to lose/use some of their tokens. Isn't there a particular phase in which we hope to enable that?

Edited my PR description to be more clear. Some variants were commented and are now completely removed (not uncommented).

Reply:
VSCode formatter changed it to that.
I can try to look at keeping one per line, indeed would be simpler to read.

Reply:
It seems the patterns we have are considered "small" by the formatting standard and are thus inlined by `cargo fmt`
https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/expressions.md#line-breaking